tests/basic/pool_function_test.go

```diff
@@ -833,3 +834,87 @@ var _ = Describe("{PoolExpandAddDiskInMaintenanceMode}", func() {
 		})
 	})
 })
+
+var _ = Describe("{PoolVolUpdateResizeDisk}", func() {
+
+	var testrailID = 34542876
+	// testrailID corresponds to: https://portworx.testrail.net/index.php?/tests/view/34542876
+
+	BeforeEach(func() {
+		StartTorpedoTest("PoolVolUpdateResizeDisk", "expand volume to the pool and pool expansion using resize-disk", nil, testrailID)
+		contexts = scheduleApps()
+	})
+	JustBeforeEach(func() {
+		poolIDToResize = pickPoolToResize()
+		log.Infof("Picked pool %s to resize", poolIDToResize)
+		storageNode, err = GetNodeWithGivenPoolID(poolIDToResize)
+		log.FailOnError(err, "Failed to get node with given pool ID")
+
+	})
+	JustAfterEach(func() {
+		AfterEachTest(contexts)
+	})
+
+	AfterEach(func() {
+		appsValidateAndDestroy(contexts)
+		EndTorpedoTest()
+	})
+	stepLog := "Expand volume to the pool and pool expansion using resize-disk"
+	It(stepLog, func() {
+		var newRep int64
+		var currRep int64
+		poolToResize = getStoragePool(poolIDToResize)
+		volSelected, err := GetVolumeWithMinimumSize(contexts, 10)
+		log.FailOnError(err, "error identifying volume")
+		opts := volume.Options{
+			ValidateReplicationUpdateTimeout: replicationUpdateTimeout,
+		}
+
+		stepLog = "Expand volume to the intended pool"
```

Comment:
Please check out example usage of "stepLog"  https://github.com/portworx/torpedo/blob/dda8caa3a1396afd1ec5c369a4a4a3b4cb2ae62c/tests/basic/pool_function_test.go#L189
If you only want to log something out, can simply call `log.InfoD("[the log message]")`.

Reply:
Done